cmd/proxy/kustomize/deployment.yaml

```diff
@@ -18,7 +18,7 @@ spec:
         image: kurl/proxy
         env:
         - name: NODE_PORT
-          value: "8800"
+          value: "30880"
```

Comment:
is this intentional?

Reply:
yes, to be compatible with microk8s dev env